src/engine/loopingcontrol.cpp

```diff
@@ -138,6 +184,18 @@ LoopingControl::LoopingControl(QString group,
     m_pCOLoopMove = new ControlObject(ConfigKey(group, "loop_move"), false);
     connect(m_pCOLoopMove, SIGNAL(valueChanged(double)),
             this, SLOT(slotLoopMove(double)), Qt::DirectConnection);
+    m_pCOLoopMoveForward = new ControlPushButton(ConfigKey(group, "loop_move_forward"));
+    connect(m_pCOLoopMoveForward, SIGNAL(valueChanged(double)),
+            this, SLOT(slotLoopMoveForward(double)));
+    m_pCOLoopMoveBackward = new ControlPushButton(ConfigKey(group, "loop_move_backward"));
+    connect(m_pCOLoopMoveBackward, SIGNAL(valueChanged(double)),
+            this, SLOT(slotLoopMoveBackward(double)));
+    m_pCOLoopMoveForwardBeatloopSize = new ControlPushButton(ConfigKey(group, "loop_move_forward_beatloop_size"));
+    connect(m_pCOLoopMoveForwardBeatloopSize, SIGNAL(valueChanged(double)),
+            this, SLOT(slotLoopMoveForwardBeatloopSize(double)));
+    m_pCOLoopMoveBackwardBeatloopSize = new ControlPushButton(ConfigKey(group, "loop_move_backward_beatloop_size"));
+    connect(m_pCOLoopMoveBackwardBeatloopSize, SIGNAL(valueChanged(double)),
+            this, SLOT(slotLoopMoveBackwardBeatloopSize(double)));
```

Comment:
@radusuciu explained the use case [above](https://github.com/mixxxdj/mixxx/pull/1187#issuecomment-280117001). I do not understand why you think these do not fit well with the GUI or controllers. The skin currently has loop_move_[forward/backward] moved to the forward/backward buttons with loop_move_[forward/backward]_beatloop_size as the right click action. Controllers do not often have loop move buttons, but mapping shift + turning a loop size encoder to loop_move_[forward/backward] is intuitive are really useful.